Shell/Parser.cpp

```diff
@@ -102,6 +102,28 @@ Vector<Command> Parser::parse()
                 m_state = State::InDoubleQuotes;
                 break;
             }
+            if(ch >= '0' && ch <= '9') {
+                if(i != m_input.length() - 1) {
+                    char next_ch = m_input.characters()[i + 1];
```

Comment:
What if the number is multi-digit, like in `foo 35>/dev/null`?

Reply:
I think that's a bashism, from my testing only `bash` supports it. None of `mksh`, `zsh`, or `dash` seem to react to it correctly.

POSIX [specifies](https://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_07) that "The largest possible value is implementation-defined; however, all implementations shall support at least 0 to 9, inclusive, for use by the application."

Reply:
Well, it'd be a nice thing to have even if not required by POSIX.

Reply:
Alright. We could also have it like in zsh, where the form looks like `command {11}>file`. I think it's nice because we avoid having ambiguities, for example should `echo $11>file` be treated like `echo ${1} 1>file` or `echo $ {11}>file`?